examples/quickstart/README.md

```diff
@@ -0,0 +1,87 @@
+# Example setup
+
+This example shows the bare minimum requirements to deploy applications on wasmCloud.
+
+It relies on the Kubernetes `default` namespace for simplicity.
+
+## Install [NATS](https://github.com/nats-io/nats-server)
+
+```bash
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+helm upgrade --install -f nats-values.yaml nats-cluster nats/nats
+```
+
+Validate installation with:
+
+```bash
+# make sure pods are ready
+kubectl get statefulset,deployment -l app.kubernetes.io/instance=nats-cluster
+```
+
+## Install wasmCloud Application Deployment Manager - [wadm](https://github.com/wasmCloud/wadm)
+
+```sh
+helm install wadm -f wadm-values.yaml --version 0.2.0 oci://ghcr.io/wasmcloud/charts/wadm
+```
+
+Validate installation with:
+
+```bash
+# make sure pods are ready
+kubectl get deploy wadm
+```
+
+## Install the operator
+
+```sh
+kubectl apply -k ../../deploy/base
+```
+
+Validate installation with:
+
+```bash
+# make sure pods are ready
+kubectl -n wasmcloud-operator get deploy
```

Comment:
Consider using `kubectl rollout status` instead:
```suggestion
kubectl rollout status deploy -l app.kubernetes.io/instance=wasmcloud-operator -n wasmcloud-operator
```

Reply:
👍 
small change here as we don't set that label
```
kubectl rollout status deploy -l app=wasmcloud-operator -n wasmcloud-operator
```